### PR TITLE
Cbo 626 fix dump script

### PR DESCRIPTION
**Pull request by willmcb**
#### What
Fix database dump script

#### Ticket

[CBO-626 fix dump script](https://dsdmoj.atlassian.net/browse/CBO-626)

#### Why
It broke when we upgraded rails to 5.2.2

Issue was that the method `arel_attributes_with_values_for_create` had become deprecated

#### How
New class that should make it easier to maintain.

--------

#### TODO (wip)

- [] test it actually works for prod
- [] possible further refactors

### CODE REVIEW

**Comment by jsugarman** [comment]
@willmcb looks good, have you tried running this against disaster env. It has a ton of old live data

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'ostruct'
```
Comment (jsugarman) [inline lib/tasks/db.rake]: is this being used?

Comment (willmcb) [reply lib/tasks/db.rake]: No - I was using it as an interim thing I'll remove.

Comment (jsugarman) [reply lib/tasks/db.rake]: cool

**Review by jsugarman — APPROVED** [review]
looks good 👍 give a go on disaster, we can merge now if you like

**Pull request merged**